features/steps/test_starter/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -3,7 +3,6 @@ flake8>=3.7.9, <4.0
 ipython>=7.31.1, <8.0
 isort~=5.0
 jupyter~=1.0
-jupyter_client>=5.1, <8.0
```

Comment:
This is a core dependence of `jupyter` so I've decided to not list it explicitly so we don't have to keep bumping the version requirements.